frontend/src/api/k8s/notebooks.ts

```diff
@@ -94,13 +96,14 @@ const assembleNotebook = (
         [KnownLabels.DASHBOARD_RESOURCE]: 'true',
       },
       annotations: {
-        'openshift.io/display-name': notebookName.trim(),
+        'openshift.io/display-name': displayName,
         'openshift.io/description': description || '',
         'notebooks.opendatahub.io/oauth-logout-url': `${origin}/projects/${projectName}?notebookLogout=${notebookId}`,
         'notebooks.opendatahub.io/last-size-selection': notebookSize.name,
         'notebooks.opendatahub.io/last-image-selection': imageSelection,
         'notebooks.opendatahub.io/inject-oauth': 'true',
         'opendatahub.io/username': username,
+        'opendatahub.io/image-display-name': displayName,
```

Comment:
Should this be the image name here? Why it's using the notebook display name?

Reply:
yep you are right. pushed a fix